ui/cap-react/src/components/search/SearchFacets.js

```diff
@@ -187,6 +187,13 @@ class SearchFacets extends React.Component {
                       isAggSelected={this.isAggSelected}
                       selectedAggs={this.props.selectedAggs}
                       onChange={category => this._onChange.bind(this, category)}
+                      deadlineDateValues={
```

Comment:
shouldn't be explicit to a key (e.g. "deadlineDateValues"). instead we should check the `meta.type` info to distinguish on facet display type